test_analysis5.py

```
@@ -0,0 +1,61 @@
+        name = os.path.basename(t).replace('.dart', '')
+        found = False
+        for root, _, files in os.walk('test'):
+            for file in files:
+                if name in file:
+                    test_file = os.path.join(root, file)
+                    with open(test_file, 'r') as f:
+                        tcount = len(f.readlines())
+                    print(f"HAS ALT TEST ({tcount} lines): {t} ({dict(sizes)[t]} lines) -> {test_file}")
+                    found = True
+                    break
+            if found: break
```

Ruff (0.15.13)

[error] Multiple statements on one line (colon)

(E701)